core/src/test/java/kafka/testkit/KafkaClusterTestKit.java

```diff
@@ -188,12 +188,13 @@ private KafkaConfig createNodeConfig(TestKitNode node) {
                     controllerNode.metadataDirectory());
             }
             props.put(SocketServerConfigs.LISTENER_SECURITY_PROTOCOL_MAP_CONFIG,
-                    "EXTERNAL:PLAINTEXT,CONTROLLER:PLAINTEXT");
+                    "EXTERNAL:PLAINTEXT,CONTROLLER:PLAINTEXT,PLAINTEXT:PLAINTEXT");
```

Comment:
why we need this change?

Reply:
The TestUtils requires sever use `PLAINTEXT` protocol, you can reference [here.](https://github.com/apache/kafka/blob/d6cd83e2fb2bab4526f07e067277b34e482f6678/core/src/test/scala/unit/kafka/utils/TestUtils.scala#L232)

Reply:
not sure why we need to call that test helper?

Reply:
In this PR, I try to minimal this patch so I don't touch any helper function.

Reply:
that is good. If you feel this PR needs some helpers, we can have another PR to address that.